cmd/test/test.go

```diff
@@ -180,15 +193,33 @@ func newTestCommand() *cobra.Command {
 						return fmt.Errorf("running test: %w", resultsErr)
 					}
 
+					var outputFile *os.File
+					if outputFilePath != "" {
+						outputFile, err = os.Create(outputFilePath)
+						if err != nil {
+							return fmt.Errorf("creating output file: %w", err)
+						}
+						defer outputFile.Close()
+					}
+
 					outputter := output.Get(runner.Output, output.Options{
 						NoColor:            runner.NoColor,
 						SuppressExceptions: runner.SuppressExceptions,
 						Tracing:            runner.Trace,
 						JUnitHideMessage:   viper.GetBool("junit-hide-message"),
+						File:               outputFile,
 					})
 					if err := outputter.Output(results); err != nil {
 						return fmt.Errorf("output results: %w", err)
 					}
+
+					if outputFilePath != "" {
+						contents, err := os.ReadFile(outputFile.Name())
+						if err != nil {
+							return fmt.Errorf("copying output file to stdout: %w", err)
+						}
+						fmt.Println(string(contents))
```

Comment:
I think we want to use `cmd.OutOrStdout()` here
```suggestion
						fmt.Fprintln(cmd.OutOrStdout(), string(contents))
```

Reply:
Done!